lib/reporting/drop_off_report.rb

```diff
@@ -375,13 +403,57 @@ def to_csvs
       end
     end
 
-    # @return [String]
-    def format_as_percent(numerator:, denominator:)
-      (100 * numerator.to_f / denominator.to_f).round(2).to_s + '%'
+    # @return [Float]
+    def percent(numerator:, denominator:)
+      (numerator.to_f / denominator.to_f).round(2)
+    end
```

Comment:
I wouldn't even include the `.round` because the pretty-printing classes will also round & format

Reply:
updated in https://github.com/18F/identity-idp/pull/9989/commits/47441e44c29aada19a63404776ad0ca2a8668d4a